runtime-modules/blog/src/lib.rs

```diff
@@ -578,36 +637,117 @@ decl_module! {
             Ok(())
         }
 
+        /// Remove reply from storage
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)` doesn't depends on the state or parameters
```

Comment:
Is the weight comment correct here?

Reply:
It was correct given the weight function used but it seemed weird so I re-run the benchmarks and now it depends on the number of replies.